include/revng/Pipeline/ContainerFactorySet.h

```diff
@@ -34,6 +34,18 @@ class ContainerFactorySet {
   const_iterator end() const { return Entries.end(); }
   size_t size() const { return Entries.size(); }
 
+  value_type &find(llvm::StringRef Name) {
+    auto Search = Entries.find(Name);
+    revng_assert(Search != Entries.end());
+    return *Search;
+  }
+
+  const value_type &find(llvm::StringRef Name) const {
+    auto Search = Entries.find(Name);
+    revng_assert(Search != Entries.end());
+    return *Search;
+  }
```

Comment:
`find` methods return an iterator. This looks more like `at()`.

Reply:
Renamed to `at`